pyaedt/modules/Mesh.py

```diff
@@ -566,50 +566,50 @@ def assign_surface_mesh(self, names, level, meshop_name=None):
         >>> from pyaedt import Hfss
         >>> aedtapp = Hfss()
         >>> o = aedtapp.modeler.create_cylinder(0, [0, 0, 0], 3, 20, 0)
-        >>> surface = aedtapp.mesh.assign_surface_mesh(o.id, 3, "Surface")
+        >>> surface = aedtapp.mesh.assign_surface_mesh(o.id,3,"Surface")
         """
-        names = self.modeler.convert_to_selections(names, True)
-        if meshop_name:
+        objects = self.modeler.convert_to_selections(objects, True)
+        if name:
             for m in self.meshoperations:
-                if meshop_name == m.name:
-                    meshop_name = generate_unique_name(meshop_name)
+                if name == m.name:
+                    name = generate_unique_name(name)
         else:
-            meshop_name = generate_unique_name("SurfApprox")
-        self.logger.info("Assigning Mesh Level " + str(level) + " to " + str(names))
-        names = self._app.modeler.convert_to_selections(names, True)
+            name = generate_unique_name("SurfApprox")
+        self.logger.info("Assigning Mesh Level " + str(level) + " to " + str(objects))
+        objects = self._app.modeler.convert_to_selections(objects, True)
 
-        if isinstance(names[0], int):
+        if isinstance(objects[0], int):
             seltype = "Faces"
         else:
             seltype = "Objects"
         props = OrderedDict(
             {
                 "Type": "SurfApproxBased",
                 "CurvedSurfaceApproxChoice": "UseSlider",
-                seltype: names,
+                seltype: objects,
                 "SliderMeshSettings": level,
             }
         )
-        mop = MeshOperation(self, meshop_name, props, "SurfApproxBased")
+        mop = MeshOperation(self, name, props, "SurfApproxBased")
         mop.create()
         self.meshoperations.append(mop)
         return mop
 
-    @pyaedt_function_handler()
-    def assign_surface_mesh_manual(self, names, surf_dev=None, normal_dev=None, aspect_ratio=None, meshop_name=None):
+    @pyaedt_function_handler(names="objects", meshop_name="name")
+    def assign_surface_mesh_manual(self, objects, surf_dev=None, normal_dev=None, aspect_ratio=None, name=None):
```

Comment:
@maxcapodi78 I'd prefer to use full names: surface_deviation instead for example.
```suggestion
    def assign_surface_mesh_manual(self, objects, surface_deviation=None, normal_dev=None, aspect_ratio=None, name=None):
```